website/pages/docs/plugins/sources/snyk/_configuration.mdx

```diff
@@ -2,15 +2,15 @@
 kind: source
 # Common source-plugin configuration
 spec:
-  name:    snyk
-  path:    cloudquery/snyk
```

Comment:
why change formatting?

Reply:
Because we don't format it this way in other plugins, and the spacing seemed inconsistent to me anyway

Reply:
(ch, meili, mssql...) - also formatted
spacing is done to align on the beginning of the values under same key

still, it's a nit